mapit/views/areas.py

```diff
@@ -394,10 +394,13 @@ def areas_by_point(request, srid, x, y, bb=False, format=''):
 
     q = query_args(request, format)
 
+    max_within = getattr(settings, 'MAPIT_WITHIN_MAXIMUM', 0)
```

Comment:
Would be good to document this in `general.yml-example`.

Reply:
I was going to add/document it in mapit.mysociety.org's conf/general.yml and pull it into its settings there, iyswim, but yes, might as well be consistent here too. Shouldn't really have any conf/general.yml stuff in this project.